fmriprep/workflows/bold/confounds.py

```diff
@@ -161,13 +161,18 @@ def init_bold_confs_wf(
 in T1w space, after their projection to the native space of each
 functional run (using the inverse BOLD-to-T1w transformation). Components
 are also calculated separately within the WM and CSF masks.
+For each CompCor decomposition, the :math:`k` components with the largest singular
```

Comment:
Sorry about this. I did not think it was on the boilerplate. Reverting...
```suggestion
For each CompCor decomposition, the k components with the largest singular
```